omnibus/package-scripts/agent/prerm

```diff
@@ -29,10 +35,9 @@ stop_agent()
         initctl stop $SERVICE_NAME-sysprobe || true
         initctl stop $SERVICE_NAME-trace || true
         initctl stop $SERVICE_NAME || true
-    elif [ "$DISTRIBUTION_FAMILY" = "Debian" ]; then
+    elif [ "$DISTRIBUTION_FAMILY" = "Debian" ] || [ "$SUSE_SYSVINIT_SUPPORT"="yes" ]; then
         if command -v service >/dev/null 2>&1; then
             service $SERVICE_NAME-process stop || true
-            service $SERVICE_NAME-sysprobe stop || true
```

Comment:
In Debian we might want to keep this line still.

Reply:
The sysprobe SysVInit script for debian was never shipped with the Agent, see #4437

Reply:
Does this only apply to SysVinit, though? Or also to upstart?

Reply:
I don't think so, as Upstart-managed debian / ubuntu systems should have been caught in the previous `initctl` block (since `initctl` is the [init control tool for Upstart](http://manpages.ubuntu.com/manpages/trusty/man8/initctl.8.html)). However, there _might_ be weird cases where a user could have upstart without `initctl`, using only `service`, or `start` / `stop` directly. I can leave this line here for now if we want to make sure we don't break any use case.

Reply:
I re-added the line, with a comment